tests/forms_tests/widget_tests/test_multiwidget.py

```diff
@@ -118,6 +123,42 @@ def test_constructor_attrs(self):
             '<input id="bar_1" type="text" class="small" value="lennon" name="name_1" />'
         ))
 
+    def test_constructor_attrs_with_type(self):
+        attrs = {'type': 'number'}
+        widget = MyMultiWidget(widgets=(
```

Comment:
I used 3 widgets to check that `attrs` is not changed inside their constructor, but now it's unnecessary.